.github/workflows/pika.yml

```
@@ -174,13 +174,17 @@
       - name: Install Deps
         run: |
           brew update
-          brew install --overwrite python@3.12 autoconf protobuf llvm wget git
-          brew install gcc@10 automake cmake make binutils
+          brew install autoconf protobuf llvm wget git python@3.13
+          brew install automake cmake make binutils
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)